src/common/veteran.js

```diff
@@ -138,7 +138,8 @@ const blankVeteran = {
   vietnamService: false,
   exposedToRadiation: false,
   radiumTreatments: false,
-  campLejeune: false
+  campLejeune: false,
+  privacyAgreementAccepted: true
```

Comment:
Should this be set to `false` first, since this is for an initial blank application?